Scripts/Python/EderNaybree.py

```diff
@@ -0,0 +1,8 @@
+from Plasma import *
```

Comment:
Please add the GPL license block as found in the other Python files. Check this as an example: https://github.com/H-uru/Plasma/blob/master/Scripts/Python/Ahnonay.py#L1:L42